### ISSUE TITLE
How to train demo without depth\normal\poses as input?

### ISSUE THREAD

**Issue by Golbstein** [cvg/nicer-slam#16]
Hello

You've claimed this is a RGB SLAM system, but as far as I can see you're using GT depths and normals to evaluate the loss of the mapping and tracking

>              ConfigTree([('assign_scale_shift_init', True),
>                          ('warp_loss_weight', 0.5),
>                          ('warp_loss_type', 'l1'),
>                          ('rgb_loss', 'torch.nn.L1Loss'),
>                          ('eikonal_weight', 0.1),
>                          ('smooth_weight', 0.005),
>                          ('depth_weight', 0.1),
>                          ('normal_l1_weight', 0.05),
>                          ('normal_cos_weight', 0.05),
>                          ('flow_weight', 0.001)])),
> 

How to modify the training on just directory with sequence of RGB images?

I also set verbose=True to print the losses and make sure I'm not mistaken

> demo_2_2024_08_30_07_54_25demo [0] (1/50): loss = 1.3251757621765137, rgb_loss = 0.1577761173248291, eikonal_loss = 0.01311636995524168, gt_depth_loss = 0.10866174101829529, warp_loss = 0.0, smooth_loss = 0.0001585051795700565, flow_loss = 0.0, psnr = 13.15900993347168

**Comment by Zzh2000** [comment]
It uses Omnidata's monocular predicted depth and normal, no gt depth and normal is needed. Please double-check the paper first. The 'gt_depth_loss' is just a variable name following MonoSDF, since we built the codebase on top of it.
